bugsnag.js

```diff
@@ -21,6 +21,7 @@ const bugsnagApiKey = (() => {
 })();
 if (bugsnagApiKey) {
   bugsnag.register(bugsnagApiKey, {
+    autoNotifyUnhandledRejection: false,
```

Comment:
Does this have to do with this PR?

Reply:
Yeah, bugsnag was getting in the way of the webRTC lib. I can push a test that illustrates.

Reply:
Ok, run `npm run serve` and then `node . http://localhost:5000/webrtc`

If you remove that bugsnag line, you'll notice it exits incorrectly.

Reply:
This is a pretty drastic change though; do we know nothing else will work?

Reply:
The other thing I tried was something like this:

```
if (bugsnagApiKey) {
  bugsnag.register(bugsnagApiKey, {
    beforeSend: function (report) {
      console.log('TKTK');
      console.log(report.errorMessage);
      // Disable reporting of errors with a given error message
      if (report.errorMessage === 'Do not report this error') report.ignore()
      // or
      if (report.errorMessage === 'Do not report this error either') return false
    },
```

But `bugsnag.register` doesn't take a `beforeSend`.

If you can figure out how to get it to work with exokit's codebase (https://docs.bugsnag.com/platforms/browsers/customizing-error-reports/) then we could ignore the specific webrtc errors.

Reply:
Could you update the PR with details on why this is impossible to fix in webrtc?

At least so we know when we can remove the hack.

Reply:
Hm, which hack?

Reply:
Using external error strings (which are subject to change) as a functionality requirement.

Reply:
Hm, then it seems there is work to do on the WebRTC side understanding what's going on here...

Reply:
Maybe, but that work can probably be deferred. Google's standard webRTC examples work. https://codelabs.developers.google.com/codelabs/webrtc-web/#5

Reply:
We don’t know if it can be deferred, because we don’t understand what’s going on.

Webrtc is full of corner cases unfortunately so Its hard to gain confidence from examples.